File: test_milestone1/repos/pure-python/src/calculator.py
"""Calculator module."""


class Calculator:
    """A simple calculator."""

    def add(self, a: float, b: float) -> float:
        return float(a + b)

    def subtract(self, a: float, b: float) -> float:
        return float(a - b)

    def multiply(self, a: float, b: float) -> float:
        result = a * b
        print(f"multiply({a}, {b}) = {result}")
        return result

    def divide(self, a: float, b: float) -> float:
        if b == 0:
            raise ValueError("Cannot divide by zero")
        return a / b


def compute_average(values: list[float]) -> float:
    """Compute the average of a list of numbers."""
    total = sum(values)
    count = len(values)
    return total / count

File: test_milestone1/repos/pure-js/src/calculator.js
// Calculator module.

class Calculator {
  add(a, b) {
    return a + b;
  }

  subtract(a, b) {
    return a - b;
  }

  multiply(a, b) {
    var result = a * b;
    console.log("multiply(" + a + ", " + b + ") = " + result);
    return result;
  }

  divide(a, b) {
    if (b === 0) {
      throw new Error("Cannot divide by zero");
    }
    return a / b;
  }
}

function computeAverage(values) {
  var total = sum(values);
  var count = len(values);
  return total / count;
}
